src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -43,32 +45,35 @@
~
~  lookupFailed = false;
~
~  filteredList$: Observable<any[]>;
~
~  recentItemsFilteredList$: Observable<any[]>;
 
   currentGeolocationPermissionGranted = false;
 
+  isDeviceLocationEnabled: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)